function/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/docker/docker v20.10.3+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/docker/go-units v0.4.0 // indirect
+	github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815 // indirect
```

Comment:
How come we have this dependency added in an unrelated module?

Reply:
Happend when running `make build`

Reply:
Sorry, seems I accidentally accepted also the changes in the function mod... will revert them

Reply:
FIXED ... mod files of fct-module were reverted